pages/sign-up.tsx

```diff
@@ -147,6 +161,18 @@ function SignUpPage(props: any) {
         />
         <aside className={styles.formAside}>Requirements: at least 8 characters, must use at least one letter and number.</aside>
 
+        {/* add confirm password */}
```

Comment:
nit: we don't need this comment, it is obvious in the naming.

Reply:
comment removed